cdap-ui/app/cdap/services/WizardStores/ArtifactUpload/ArtifactUploadStore.js

```diff
@@ -81,7 +73,7 @@ const upload = (state = defaultUploadState, action = defaultAction) => {
   }
 
   return Object.assign({}, stateCopy, {
-    __complete: isComplete(stateCopy, configurationStepRequiredFields)
+    __complete: true
```

Comment:
This cannot be true right? It still needs the jar before enabling the finish button?